zipline/testing/fixtures.py

```diff
@@ -177,7 +177,14 @@ class WithAssetFinder(object):
     def _make_info(cls):
         return None
 
-    make_equities_info = _make_info
+    @classmethod
+    def make_equities_info(cls):
+        return make_simple_equity_info(
+            cls.get_sids(),
```

Comment:
It's coming from `EventLoaderCommonMixin` in `base.py`, which is probably not the best way to organize this. A better way to organize this might be to have `EventLoaderCommonMixin` be turned into a fixture which subclasses from `WithAssetFinder` and to have `make_equities_info` be defined there instead.